_docs/checker-workflows.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Checker Workflows
+permalink: /docs/publisher-tutorials/checker-workflows/
+---
+
+# Checker Workflows
+
+## Tutorial Goals
+* Learn about checker workflows
+* Add a checker workflow to an existing tool
+* Update a checker workflow
+* Launch a checker workflow
+
+## Overview Checker Workflows
+Checker workflows are additional workflows you can associate with a tool or workflow. The purpose of them is to ensure that a tool or workflow, given some inputs, produces the expected outputs. Below is a visual overview of how a checker workflow looks.
```

Comment:
Maybe "produces the expected outputs on a platform different from the one where you are developing. 
Checker workflows can be used in the near future for iterations of the GA4GH-DREAM challenge and in the long-term picked up and run by the automated GA4GH workflow "testbed" to test your workflows across a variety of workflow platforms.  See this presentation https://docs.google.com/presentation/d/1VXdReGYXayzO7Jr-9XaLHNv6Wt46CwfvkfFDR8OEgJM/edit?usp=sharing for more details and contact the [GA4GH Cloud Work Stream](https://github.com/ga4gh/wiki/wiki) for more information on contributing to the workflow testbed or related APIs.